src/esbmc/options.cpp

```diff
@@ -84,6 +84,7 @@ const struct group_opt_templ all_cmd_options[] = {
     {"document-subgoals", NULL, "generate subgoals documentation"},
     {"no-arch", NULL, "don't set up an architecture"},
     {"no-library", NULL, "disable built-in abstract C library"},
+    {"output-goto", NULL, "writing goto program to binary file"},
```

Comment:
I don't like this description. Maybe "export generated goto program" would be better.

Reply:
Thanks for your correction!